draft-ietf-tls-tls13.md

```diff
@@ -3366,11 +3366,21 @@ from the resumption master secret:
 The client MAY use this PSK for future handshakes by including
 the ticket value in the "pre_shared_key" extension in its ClientHello
 ({{pre-shared-key-extension}}) and supplying a suitable PSK cipher
-suite. Servers may send multiple tickets on a single connection, for
-instance after post-handshake authentication. For handshakes that
+suite. For handshakes that
 do not use a resumption_psk, the resumption_context is a string of
 L zeroes.
 
+The NewSessionTicket message may include multiple ticket
+labels bound to the same resumption_psk and context; this allows
+a client to initiate multiple concurrent connections in parallel
+without exposing themselves to tracking of the PSK label, which
+is carried in the clear. Servers MAY send multiple NewSessionTicket
+messages on a single connection, for
+instance after post-handshake authentication. When a client
+receives a NewSessionTicket message it SHOULD discard any existing
```

Comment:
If we have this text, then a server should not send multiple NewSessionTickets in close succession, as the removed text would have had it do.  Do we want to say anything about the frequency of sending NewSessionTicket?  I am inclined to say that we do not, since it would depend on the use case for the given connection.